packages/harpguru-core/src/components/menu-close-button/menu-close-button.tsx

```diff
@@ -1,15 +1,36 @@
-import { TapGestureHandler } from 'react-native-gesture-handler'
+import { useTimingTransition } from 'react-native-redash'
+import Animated, { Easing, interpolate } from 'react-native-reanimated'
+import { TapGestureHandler, State } from 'react-native-gesture-handler'
+import type { TapGestureHandlerStateChangeEvent } from 'react-native-gesture-handler'
 import { StyleSheet, View } from 'react-native'
 import React from 'react'
 import { AntDesign } from '@expo/vector-icons'
 
 import type { MenuProps } from '../../types'
 import { getSizes, colors } from '../../styles'
+import { tapAnimationDuration } from '../../constants'
```

Comment:
This needs to either be used more consistently or there needs to be a root level custom hook which abstracts away the calculations of the animation values.

Reply:
They are all very similar after all. I think  any inconsistencies are caused by inconsistencies in how the tap handlers which drive the animations are set upl

Reply:
Done